src/containers/ansible-repository/tab-details.tsx

```diff
@@ -29,18 +26,6 @@ export const DetailsTab = ({ item }: TabProps) => {
           label: t`Retained version count`,
           value: item?.retain_repo_versions ?? t`All`,
         },
-        {
-          label: t`Distribution`,
-          value: <LazyDistributions repositoryHref={item?.pulp_href} />,
```

Comment:
Would we not want to show associated distributions & url?

Reply:
There is technically already a distributions tab that will show associated distros, so I felt that was redundant. As for the repo url, that is actually a Galaxy thing. Although maybe I could be sold on its convenience, to encourage users to create 1-to-1 mappings between their repositories and distributions, to make operating Pulp simpler.

Reply:
Ooh, right, I remember now, base path of a first distribution associated with the repo, unless there's an identically named distribution :)

Yeaah, maybe let's keep it if it works and see if users find it useful? Maybe everybody uses distributions manually, but the repo=repo version=distribution abstraction definitely made things simpler :)